sdk/avs/Azure.ResourceManager.Avs/tests/AvsManagementTestBase.cs

```diff
@@ -37,5 +47,29 @@ protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionReso
             var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
             return lro.Value;
         }
+
+        protected async Task<AvsPrivateCloudResource> getAvsPrivateCloudResource()
+        {
+            string subscriptionId = DefaultSubscription.Data.SubscriptionId;
+            string resourceGroupName = "avs-sdk-test";
```

Comment:
why do we have the resource group name hard-coded? is it because it was created beforehand?

Reply:
Yes, it was created before hand and locked for sdk testing.